### PR TITLE
add(valkey): metrics in prometheus

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/DDB-1373

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

File: docs/products/valkey/reference/valkey-metrics-in-prometheus.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+Replace `USERNAME:PASSWORD` with your Prometheus credentials and `PROMETHEUS_URL`
+with the Service URI from the **Connection information** section.
+
+</TabItem>
+</Tabs>
+
+<HostMetrics />
+
+## Valkey-specific metrics
```
Comment (bogdanp05) [inline docs/products/valkey/reference/valkey-metrics-in-prometheus.md:50]: Why don't we just link directly to the telegraf redis plugin? We wouldn't have to worry about keeping this page in sync.

Comment (wojcik-dorota) [reply docs/products/valkey/reference/valkey-metrics-in-prometheus.md:50]: @bogdanp05 thanks! Added the link :-)

**Review by bogdanp05 — APPROVED** [review]
lgtm

**Review by staceysalamon-aiven — APPROVED** [review]
LGTM

**Pull request merged**